docker-app/Dockerfile

```diff
@@ -12,14 +12,20 @@ ARG PIP_DISABLE_PIP_VERSION_CHECK=1
 ARG PIP_NO_CACHE_DIR=1
 
 # install psycopg2 requirements
-RUN apt-get update && apt-get install -y libpq-dev python3-dev gcc
+RUN apt-get update  \
+    && apt-get install -y  \
+    libpq-dev  \
+    python3-dev  \
+    gcc \
+    && rm -rf /var/lib/apt/lists/*
 
 # install `pip-compile` (as part of `pip-tools`)
 RUN pip install pip-tools
 
 # install pip dependencies
 COPY ./requirements/requirements.txt /requirements/requirements.txt
-RUN pip3 install -r requirements/requirements.txt
+RUN pip3 install -r requirements/requirements.txt \
+    && rm requirements/requirements.txt
 
 # save 60 MB of data... Safe to delete according to botocore contributor https://github.com/boto/botocore/issues/1629#issuecomment-451309885
 RUN ls -Q /usr/local/lib/python3.10/site-packages/botocore/data | grep -xv "endpoints.json" | xargs rm -rf
```

Comment:
Where do we want to save the storage? As this is not done within the same layer, the data is still on local systems. This would only matter for images which are not used as far as I know.

Reply:
Github CI.